### PR TITLE
[CSL-531] Build Windows uninstaller for signing and fix certification failure

### PR DESCRIPTION
**Pull request by jmitchell**

### CODE REVIEW

File: installers/WindowsInstaller.hs [hunk 1]
```diff
@@ -10,25 +10,24 @@ import           Turtle             (echo, procs)
...
-      "--update-latest-path", "\"" <> installerPath <> "\"",
-      "--logs-prefix", "\"%APPDATA%\\Daedalus\\Logs\"",
-      "--db-path", "\"%APPDATA%\\Daedalus\\DB-0.2\"",
-      "--wallet-db-path", "\"%APPDATA%\\Daedalus\\Wallet-0.2\"",
-      "--wallet"
-      ] <> ("--peer" : (L.intersperse "--peer" ipdht))
+  [ "--node \"%PROGRAMFILES%\\Daedalus\\cardano-node.exe\""
+  , "--node-log-path", "\"%APPDATA%\\Daedalus\\Logs\\cardano-node.log\""
+  , "--wallet \"%PROGRAMFILES%\\Daedalus\\Daedalus.exe\""
+  , "--updater C:/TODO" -- TODO
```
Comment (jmitchell) [inline installers/WindowsInstaller.hs]: Messed up the merge here. I'll add back the appropriate `--updater` soon.

File: installers/WindowsInstaller.hs [hunk 2]
```diff
@@ -10,25 +10,24 @@ import           Turtle             (echo, procs)
...
-    , "--updater \"" <> installerPath <> "\""
-    , "--node-timeout 5"
-    , (" -n " ++ (L.intercalate " -n " nodeArgs))
-    ]
-  where
-    installerPath = "%APPDATA%\\Daedalus\\Installer.exe"
-    nodeArgs = [
-      "--listen", "0.0.0.0:12100",
-      "--keyfile", "\"%APPDATA%\\Daedalus\\Secrets\\secret.key\"",
-      "--update-latest-path", "\"" <> installerPath <> "\"",
```
Comment (jmitchell) [inline installers/WindowsInstaller.hs]: Also clobbered this arg.

**Comment by jmitchell** [comment]
All Windows App Certification Kit tests, except the UAC test, are now passing.

The UAC test fails with the following message. I expect it will pass once we make an installer with a signed version of the packaged `uninstall.exe`. @domenkozar, let me know if you'd like any help with that part. At the very least I can rerun WACK on a manually signed installer+uninstaller.

> Main executable C:\Program Files\Daedalus\uninstall.exe is marked as either "RequiredAdministrator" or "HighestAvailable" and is not signed.

**Comment by domenkozar** [comment]
Awesome :) I'l merge this and add my signing automation.

**Pull request merged**